docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -182,139 +224,85 @@ Teleport > Connecting to prod.teleport.example.com over SSH
 
-### Filters
-
-Filter expressions allow for more detailed control over the scope of an allow
-policy or require policy.
-
-Require policies can specify which users they consider as valid with a filter
-expression. The filter context has a `user` object defined with the set fields
-`roles` and `name`.
-
-Here is an example of a filter expression that evaluates to true if the user is
-Adam or if the user has the trait `cs-observe`:
-
-```
-equals(user.name, "adam") || contains(user.spec.roles, "cs-observe")
-```
-
-A filter expression is a string statement used to define logic based on a set of
-input variables. The filter expressions follow a restricted subset of Go syntax
-and supports the following functions and operators:
-
-- `contains(set, item)`: Returns true if the item is in the set, otherwise
-  false. The set can be a string or an array.
-- `equals(a, b)`: Returns true if the two values are equal, otherwise returns
-  false.
-- `![expr]`: Negates a boolean expression.
-- `[expr] && [expr]`: Performs a logical AND on two boolean expressions.
-- `[expr] || [expr]`: Performs a logical OR on two boolean expressions.
-
-### Session kinds
-
-Require and allow policies have to specify which sessions they apply to. Valid
-options are `ssh` and `k8s`.
-
-- `ssh` policies apply to all SSH sessions on a node running the Teleport SSH server.
-- `k8s` policies apply to all Kubernetes sessions on clusters connected to Teleport.
-
 ### Participant modes
 
 A participant joining a session will always have one of three modes:
 
-- `observer`: Allows read-only access to the session. You can view output but cannot control the session in any way nor send any input.
-- `moderator`: Allows you to watch the session. You can view output and forcefully terminate or pause the session at any time, but can't send input.
+- `observer`: Allows read-only access to the session. You can view output but 
+  cannot control the session in any way nor send any input.
+- `moderator`: Allows you to watch the session. You can view output and forcefully 
+  terminate or pause the session at any time, but can't send input.
 - `peer`: Allows you to collaborate in the session. You can view output and send input.
 
-When joining a session with `tsh join` or `tsh kube join`, a user can specify a
-participant mode with the `--mode <mode>` flag , where the mode is one of `peer`,
-`moderator` or `observer`. By default, the mode is `observer`.
-
-A participant may leave a session with the shortcut `^c` (Control + c) while in observer or
-moderator mode. When in moderator mode, a participant may also forcefully
-terminate the session at any point in time with the shortcut `t`.
-
-### Require policy count
+If you join a session with `tsh join` or `tsh kube join`, you can specify a
+participant mode with the `--mode <mode>` command-line option, where `<mode>` is `peer`,
+`moderator`, or `observer`. The default participant mode is `observer`.
```

Comment:
Do note that defaulting to `observer` is a relatively recent change, so be careful when backporting this. (Off the top of my head, I think it only applies to v13 and up)